contents/docs/experiments/new-outline.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: New outline (do not merge)
+---
+
+## Experiments
+
+- Overview
+	- Keep as-is for now
+- Installation
+- Start here
+	- New doc with the key concepts you need to understand
+	- Outline:
+
+## Common use-cases
+
+- Click rate
+	- Trend with count
+- Click-through rate
+	- Funnel
+- Session length
+	- Trend with property value
+- Retention ?
+
+## Setting up
+
+- [Creating an experiment](/docs/experiments/creating-an-experiment)
+	- Refresh doc based on new UI
+	- Outline:
+		- Navigate to Experiments and click "New experiment"
+		- Explanation of each form field:
+			- Name
+			- Feature flag key
+			- Description
+			- Participant type
+			- Variants
+			- Holdout group
+				- Link to holdouts documentation
+- Defining experiments with code
+- Designing no-code experiments
+- [Testing and launching](/docs/experiments/testing-and-launching)
+	- Keep the first part of the existing doc.
+	- Potentially rename to "Starting the experiment"
+- Evaluating results
+	- Use the second part of the existing 'Testing and launching' doc.
+
+## Features
+
+- [Experiments without feature flags](/docs/experiments/running-experiments-without-feature-flags)
+	- Should this move to Guides?
+- [Holdouts](/docs/experiments/holdouts)
+	- Keep existing doc
+
+## Methodology
+
+- Statistics primer
+- Trends with count
+- Trends with property value
+- Funnels
+	- Attribution type: Funnels allow you to have the attribution assigned at any point in the funnel, not just the first step. If the user assigned to a variant later on in the funnel, then they'll be included in the entire funnel.
+- [Traffic allocation](/docs/experiments/traffic-allocation)
+- [Sample size and running time](/docs/experiments/sample-size-running-time)
```

Comment:
Not entirely sure what we talk about in these sections? But in general I think its a good direction
```suggestion
## Under the hood

- Statistics primer
- Trends with count
- Trends with property value
- Funnels
	- Attribution type: Funnels allow you to have the attribution assigned at any point in the funnel, not just the first step. If the user assigned to a variant later on in the funnel, then they'll be included in the entire funnel.
- [Traffic allocation](/docs/experiments/traffic-allocation)
- [Sample size and running time](/docs/experiments/sample-size-running-time)
```